cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -766,7 +812,7 @@ protected String getServiceAccountName() {
             }
 
             if (listeners.getExternal() != null) {
-                varList.add(buildEnvVar(ENV_VAR_KAFKA_EXTERNAL_ENABLED, "TRUE"));
+                varList.add(buildEnvVar(ENV_VAR_KAFKA_EXTERNAL_ENABLED, listeners.getExternal().getType()));
```

Comment:
We are not bringing a boolean value anymore. Should we rename it in something like `ENV_VAR_KAFKA_EXTERNAL_TYPE`?

Reply:
Well, its used for both, enabling as well as type. If it is not specified, it means the external listener is disabled. So TBH I'm not sure its worth the renaming.

Reply:
Well in my mind ENABLED brings a boolean and not a type value while TYPE is clear; at same time empty TYPE makes still sense as disabling external. Btw I can leave with it if you love ENABLED much more. :)

Reply:
Its not that I love it ... I just do not want to re-test everything again. Its not just simple refactoring in Idea because you need to change the Docker images etc. :-o